README.md

```diff
@@ -1,11 +1,22 @@
 [![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
 
+## Dependencies
+
+The `opentracing-spring-jaeger-starter` simply contains the code needed to provide a Jaeger implementation of the OpenTracing's `io.opentracing.Tracer`
+interface.
+
+For a project to be able to actually instrument a Spring stack, one or more of the purpose built starters (like `io.opentracing.contrib:opentracing-spring-web-starter` or `io.opentracing.contrib:opentracing-spring-cloud-starter`)  
+would also have to be included in the POM.
+
+The `opentracing-jaeger-spring-web-starter` starter is convenience starter that includes both `opentracing-spring-jaeger-starter` and `opentracing-spring-web-starter`
```

Comment:
`opentracing-spring-jaeger-web-starter` seems better to me. All spring related deps starts with `openracing-spring`

Reply:
I also believe what you propose is the appropriate name

Reply:
@cmoulliard Do you agree we change the module name to `opentracing-spring-jaeger-web-starter` to fit the naming convention used throughout the other opentracing projects?

Reply:
I agree as well.

Reply:
I agree too if we will also follow the same convention to name the following starter `opentracing-spring-zipkin-starter` if we will create it in the future too

Reply:
@cmoulliard Yes, that's the plan :wink: